modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -841,12 +843,16 @@ Setup a Cron job or similar with something like the following two commands:
 sudo rm -f /opt/oc/store1 # With a different schedule
 ----
 
-The first run will create the `/opt/oc/store1` with the serialized storages, the rest of the executions
-will use that file. The second Cron job, the one removing the file, will force the `wnd:process-queue` to
+The first run will create the `/opt/oc/store1` with the serialized storages, the rest of the executions will use that file. The second Cron job, the one removing the file, will force the `wnd:process-queue` to
 refresh the data.
 
-It's intended to be run in a different schedule, so there are several
-executions of the `wnd:process-queue` fetching the data from the file.
-Note that the file can be removed manually at any time if it's needed
-(for example, the admin has reset some passwords, or has been notified
-about password changing).
+It's intended to be run in a different schedule, so there are several executions of the `wnd:process-queue` fetching the data from the file. Note that the file can be removed manually at any time if it's needed (for example, the admin has reset some passwords, or has been notified about password changing).
```

Comment:
```suggestion
It's intended to be run in a different schedule, so there are several executions of the `wnd:process-queue` fetching the data from the file. Note that the file can be removed manually at any time if it's needed (for example, in case the admin has reset some passwords or has been notified about password changes).
```